docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -12,7 +12,22 @@ Each component in a single devfile must have a unique name and use one of the ob
 |container
 |componentObject
 |no
-|The list of containers that you use.
+|The list of containers.
+
+|kubernetes
+|componentObject
```

Comment:
If the entries in these two columns can be in monospace, I'd prefer them monospaced for readability as a reader.